css/style.css

```diff
@@ -0,0 +1,503 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Serif:wght@400;700&family=Roboto&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  box-sizing: border-box;
+}
+
+.wrap__site {
+  min-width: 375px;
+  max-width: 1920px;
+  width: 100%;
+  height: 100vh;
+  margin: 0 auto;
+  display: flex;
+}
+
+/* header */
+.header__burger {
+  display: none;
+}
+
+.header {
+  display: flex;
+  height: 35px;
+  max-width: 1550px;
+  justify-content: space-between;
+  margin: 41px auto 168px;
+}
+
+.header_logo-icon {
+  display: flex;
+}
+
+.logo-icon__item {
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  opacity: 0.7;
+}
+
+.logo-icon__item-blue {
+  position: relative;
+  z-index: 1;
+  background-color: #0056ec;
+  left: 23px;
+}
+
+.logo-icon__item-white {
+  position: relative;
+  z-index: 2;
+  background-color: #c8dbfb;
+  top: 8px;
+  left: 4px;
+}
+
+.header__logo {
+  font-family: 'PT Serif', serif;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  padding: 0 0 0 12px;
+}
+
+.header__menu {
+  display: flex;
+  width: 383px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.menu__item {
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 21px;
+  color: #000;
+  display: inline-block;
+}
+
+.menu__item::after {
+  display: block;
+  content: "";
+  height: 3px;
+  width: 0%;
+  background-color: #0056ec;
+  transition: 0.4s ease-in-out;
+}
+
+.menu__item:hover::after,
+.menu__item:focus::after {
+  width: 100%;
+}
+
+.menu__item_active::after {
+  display: block;
+  content: "";
+  height: 3px;
+  width: 100%;
+  background-color: #0056ec;
+}
+
+/* content */
+.content {
+  display: flex;
+  max-width: 1518px;
+  height: 659px;
+  margin: 0 auto;
+  margin-bottom: 131px;
+  flex-direction: row;
+}
+
+.content__wrap {
+  height: 445px;
+  max-width: 567px;
+  align-items: center;
+  margin-right: 203px;
+  padding: 27px 0 0 0;
+}
+
+.content__title {
+  max-width: 567px;
+  height: 255px;
+  font-family: 'PT Serif', serif;
+  font-weight: bold;
+  font-size: 66px;
+  line-height: 87px;
+}
+
+.content__title > .content__title_gray {
+  background-color: #ecf3ff;
+  padding: 2px 0 2px 0;
+  line-height: 1.6;
+}
+
+.content__desc {
+  max-width: 455px;
+  height: 52px;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 26px;
+  margin-bottom: 40px;
+}
+
+.content__bt {
+  max-width: 270px;
+  height: 57px;
+  background-color: #0056EC;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5593059)